public/styles/common.css

```diff
@@ -17,21 +17,21 @@ body {
   font-style: normal;
   font-weight: 400;
   src: local("Roboto Regular"), local("Roboto-Regular"),
-    url("../Roboto-Regular.ttf");
+    url("./Roboto-Regular.ttf");
```

Comment:
Do we need the `./` at all? Does `url("Roboto-Regular.ttf");` work?

Reply:
removed ./ and fonts working well on local server.